package.json

```diff
@@ -38,6 +38,7 @@
 		"checks:fix": "fluid-build --task checks:fix",
 		"ci:build": "fluid-build --task ci:build",
 		"ci:build:docs": "fluid-build --task ci:build:docs",
+		"ci:test:azureclient:tinylicious": "pnpm run -r --no-sort --stream --no-bail test:azureclient:tinylicious:report",
```

Comment:
Rather than running this recursively, maybe target it specifically at the azure client end-to-end tests and the command you want?

Reply:
I think that makes sense, but I was aiming to replicate how we run our other e2e test suite for consistency